src/components/SharedFields/SharedOptions.tsx

```diff
@@ -0,0 +1,101 @@
+const SharedOptions = (
+  optionPicked: string
+): { label: string; value: string }[] | undefined => {
+  const genderOptions = [
```

Comment:
```suggestion
  const GENDER_OPTIONS = [
```

Keeping in mind my comment about exporting the options individually, we should update the naming of all of these variables. Although `const` can technically be altered, we don't intend it to be altered. Because of this, we should use a constant naming scheme for these variables, i.e. `const VARIABLE_NAME`.